README.md

```diff
@@ -106,16 +124,26 @@ async def basic_transcribe():
         # NOTE: For pre-recorded files longer than 5 minutes, the sent audio
         # chunks should be rate limited to match the realtime bitrate of the
         # audio stream to avoid signing issues.
-        async with aiofile.AIOFile('tests/integration/assets/test.wav', 'rb') as afp:
-            reader = aiofile.Reader(afp, chunk_size=1024 * 16)
+        start_time = time.time()
+        total_audio_sent = 0
+
+        async with aiofile.AIOFile(AUDIO_PATH, "rb") as afp:
+            reader = aiofile.Reader(afp, chunk_size=CHUNK_SIZE)
             async for chunk in reader:
                 await stream.input_stream.send_audio_event(audio_chunk=chunk)
+                total_audio_sent += len(chunk) / (
+                    BYTES_PER_SAMPLE * SAMPLE_RATE * CHANNEL_NUMS
+                )
+                # sleep to simulate real-time streaming
+                wait_time = start_time + total_audio_sent - time.time()
+                await asyncio.sleep(wait_time)
```

Comment:
We can probably move this out into a general helper function that takes a stream and a reader. It can apply the delays for us and avoids have this all inlined.

e.g.

```python
async def apply_realtime_delay(stream: AudioStream, reader: aiofile.Reader) -> None:
    start_time = time.time()
    total_audio_sent = 0
    async for chunk in reader:
        await stream.input_stream.send_audio_event(audio_chunk=chunk)
        total_audio_sent += len(chunk) / (
            BYTES_PER_SAMPLE * SAMPLE_RATE * CHANNEL_NUMS
        )
        # sleep to simulate real-time streaming
        wait_time = start_time + total_audio_sent - time.time()
        await asyncio.sleep(wait_time)
```

Then the `write_chunks` code can be simplified to something like:

```python 
    async with aiofile.AIOFile(AUDIO_PATH, "rb") as afp:
        reader = aiofile.Reader(afp, chunk_size=CHUNK_SIZE)
        await apply_realtime_delay(stream, reader)
```

We may even just add apply_realtime_delay to a utils file in the SDK itself. This can be imported in the example files and abstracts away some of the complexity that a new user may not care about.